readthedocs/builds/models.py

```diff
@@ -615,7 +616,14 @@ class Build(models.Model):
         choices=BUILD_STATE,
         default='finished',
     )
-    status_code = models.BooleanField(_('Status code'), null=True, default=None, blank=True)
+    status_code = models.CharField(
```

Comment:
I don't think `code` is really the right word for this, since that usually means a number (like `exit_code`). I think `status` is probably fine.

I worry this is likely going to confuse things a bit with our `state` field. We should explain a bit more in a comment at least the vision behind how this field is different than `state`.

Reply:
OK. I will use `state` and copy what Anthony wrote in the other PR about their differences.